app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Idv::HowToVerifyController < ApplicationController
```

Comment:
style, I think we typically prefer to use the namespaced references, ex
```suggestion
module Idv
  class HowToVerifyController < ApplicationController
```